Mathlib/Order/RelSeries.lean

```diff
@@ -692,6 +690,21 @@ def mk (length : ℕ) (toFun : Fin (length + 1) → α) (strictMono : StrictMono
   toFun := toFun
   step i := strictMono <| lt_add_one i.1
 
+/-- An injection from the type of strictly monotone functions with limited length to `LTSeries`. -/
+def inj (n : ℕ) : {f : (l : Fin n) × (Fin (l + 1) → α) // StrictMono f.2} ↪ LTSeries α where
```

Comment:
This name is maybe a bit too generic? What do you think of `injStrictMono` or something like that?

Reply:
I've changed it to `injStrictMono`.